Domain/ProductManagement/Product.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LenaInventoryManagementSystem.Domain.ProductManagement
+{
+    public class Product
+    {
+        private string _productName;
+        private double _price;
+        private int _quantity;
+        private int _maxItemsInStock;
+
+        public Product()
+        {
+            _maxItemsInStock = 150;
+        }
+
+        public Product(string name)
+        {
+            _productName = name;
+        }
+
+        public Product(string name, double price)
+        {
+            _productName = name;
```

Comment:
you don't have to set the name again here, you already have a constructor that sets the name alone, try to call that construct instead here.